packages/cli/src/commands/integration/add.ts

```diff
@@ -265,14 +267,29 @@ async function provisionResourceViaCLI(
     return 1;
   }
 
-  return provisionStorageProduct(
-    client,
-    product,
-    installation,
-    name,
-    metadata,
-    billingPlan
-  );
+  try {
+    const authorizationId = await getAuthorizationId(
+      client,
+      teamId,
+      installation,
+      product,
+      metadata,
+      billingPlan
+    );
+
+    return provisionStorageProduct(
```

Comment:
Either do `await` here or move this call outside of `try/catch`, but IMHO it leads to bugs when `return retPromise()` is called inside the try/catch that has no chance of catching it.

Reply:
Awaiting now 👍